GitCommands/Git/GitModule.cs

```diff
@@ -1890,6 +1886,18 @@ private static void UpdateIndex(StreamWriter inputWriter, string filename)
             inputWriter.BaseStream.Write(bytes, 0, bytes.Length);
         }
 
+        public GitArgumentBuilder UpdateIndexCmd(bool showErrorsWhenStagingFiles)
```

Comment:
Please make it `private`, we don't want to increase the public surface of this class.
You can test the method via a [`TestAccessor`](https://github.com/gitextensions/gitextensions/search?q=testaccessor&unscoped_q=testaccessor)

Reply:
Sorry, the "`TestAccessor` pattern" was the missing piece that I didn't fully understand. Thus my "workaround" of making the method public.
I think I got it right this time (or at least I'm getting closer :smirk: ). Please check again.